src/clarity-angular/layout/vertical-nav/_vertical-nav.clarity.scss

```diff
@@ -0,0 +1,279 @@
+
+            //Others
+            line-height: $vertical-nav-item-height;
+            outline-offset: -5px;
+
+            @include vertical-nav-link-colors();
+
+            &:hover .nav-icon {
+                fill: $clr-action-blue;
+            }
+        }
+
+        .nav-link {
+            flex: 0 0 $vertical-nav-item-height;
+        }
+
+        //Parent .nav-group
+        .nav-group-trigger {
+            //Dims
+            width: 100%;
+
+            //Others
+            text-align: left;
+        }
+
+        //Parent .nav-trigger
+        .nav-trigger-icon {
+            //Display
+            align-self: center;
+
+            //Others
+            transition: all 0.2s ease-in-out;
+        }
+
+        .nav-header {
+            //Dims
+            padding: 0 $vertical-nav-link-padding-right 0 $vertical-nav-link-padding-left;
+
+            //Others
+            @include clr-getTypePropertiesForDomElement(vertical_nav_header, (font-size, font-weight, letter-spacing));
+            line-height: $vertical-nav-item-height;
+        }
+
+        //Parent .nav-link
+        .nav-text {
+            //Display
+            flex: 1 1 auto;
+
+            //Mixins
+            @include overflow-ellipsis();
+        }
+
+        //Parent .nav-text
+        .nav-icon {
+            //Display
+            flex: 0 0 $vertical-nav-icon-size;
+            align-self: center;
+
+            //Dims
+            height: $vertical-nav-icon-size;
+            width: $vertical-nav-icon-size;
+            margin-right: $vertical-nav-icon-right-margin;
+
+            //Others
+            vertical-align: middle;
+        }
+
+        //Parent .nav-group-trigger
+        .clr-nav-caret {
+            $caret-left-padding: $vertical-nav-link-padding-right;
+
+            //Display
+            flex: 0 0 $vertical-nav-icon-size + $caret-left-padding;
+            align-self: center;
+
+            //Dims
+            margin-left: auto;
+            padding-left: $caret-left-padding;
+
+            //Others
+            vertical-align: middle;
+
+            clr-icon {
+                transition: all 0.2s ease-in-out;
+            }
+        }
+
+        //Interactions
+        &.has-nav-groups {
+            .nav-link,
+            .nav-group .nav-group-trigger {
+                font-weight: 600;
+            }
+
+            .nav-group .nav-link {
+                font-weight: normal;
+            }
+        }
+
+        &.has-icons {
+            .nav-group-children .nav-link {
+                padding-left: $vertical-nav-icon-size + $vertical-nav-icon-right-margin + $vertical-nav-link-padding-left;
+            }
+        }
+
+        .main-container:not([class*="open-overflow-menu"]):not([class*="open-hamburger-menu"]) &.is-collapsed {
+            //Dims
+            width: $vertical-nav-min-width;
+            min-width: $vertical-nav-min-width;
+
+            //Others
+            cursor: pointer;
+
+            .nav-trigger {
```

Comment:
are you talking about this one @youdz: `.main-container:not([class*="open-overflow-menu"]):not([class*="open-hamburger-menu"]) &.is-collapsed {`. I don't really see an alternative to this :-(. Open to change this to something shorter if anyone has any better ideas?